$MirOS: ports/mailnews/pine/patches/patch-pico_pilot_c,v 1.2 2005/05/04 23:37:40 tg Exp $
--- pico/pilot.c.orig	Fri Jun 11 21:49:18 2004
+++ pico/pilot.c	Fri Oct  7 19:12:53 2005
@@ -38,7 +38,7 @@ static char rcsid[] = "$Id: pilot.c 1370
 #include	"headers.h"
 
 
-#define		PILOT_VERSION	"UW PILOT 2.0"
+#define		PILOT_VERSION	"UW PILOT 2.0L"
 
 
 extern char *gethomedir();
